src/main/java/net/vulkanmod/config/VideoResolution.java

```diff
@@ -77,6 +81,13 @@ private static int determineDisplayServer() {
         };
     }
 
+
+    private static String determineDE() {
+        String xdgSessionDesktop = System.getenv("XDG_SESSION_DESKTOP");
+        return xdgSessionDesktop !=null ? xdgSessionDesktop : "NONE";
```

Comment:
Probably should fix the spacing error here.

Reply:
Tbh VulkanMod in general has alot of formatting issues like this, not just this small pr.

We don't always prioritise formatting as we are busy with other stuff.

But if bothers ppl I can suggest to Collateral to do a formatting/syntax cleanup at some point if it becomes a big issue.

Reply:
Have corrected the formatting via a forced push, hope that's OK.